docs/PPO_FIXES_APPLIED.md

```
@@ -0,0 +1,327 @@
+MIN_ADV_STD = 1e-8         # Min std for advantage normalization
+LOGIT_SCALE = 10.0         # Bounded logit output range
+```
+
+---
+
+## Architecture Changes
+
+### ActorCritic Network
+
+**BEFORE:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)